js/index.js

```diff
@@ -1,23 +1,48 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
+  // console.log('Calculating subtotal, yey!');
+  // console.log(product)//.querySelector('.price span').innerText)
   //... your code goes here
+  const price = product.querySelector('.price span')
+  // console.log(price)
+  const quantity = product.querySelector('.quantity input')
+  // console.log(quantity.value)
+  const subtotal = product.querySelector('.subtotal span')
+  // console.log(subtotal)
+  subtotal.innerHTML = price.innerHTML * quantity.value
+  // NICEEE
+  const total = product.querySelector('#total-value span')
+  // console.log(total)
+
 }
 
 function calculateAll() {
   // code in the following two lines is added just for testing purposes.
   // it runs when only iteration 1 is completed. at later point, it can be removed.
   const singleProduct = document.querySelector('.product');
   updateSubtotal(singleProduct);
+  const allSubs = document.querySelector('.subtotal span')
```

Comment:
Good work up to now! Just some code lines missing to update the total value with the value from each product ;) Try to add that if you find the time!